### PR TITLE
Add hook for revoking a ppl

### PR DESCRIPTION
**Pull request by wheelsandcogs**
~Waiting on https://github.com/UKHomeOffice/asl-resolver/pull/99~

### CODE REVIEW

File: lib/hooks/create/project.js [hunk 1]
```diff
@@ -26,9 +28,28 @@ module.exports = settings => {
...
+              // Project revoked by establishment user - needs inspector review
+              return model.setStatus(withInspectorate.id);
+            }
+
+            if (profile.asruLicensing) {
+              // Project revoked by licensing, does not need review
+              return model.setStatus(autoResolved.id);
+            }
+
+            // Project revoked by inspector - needs licensing review
```
Comment (joefitter) [inline lib/hooks/create/project.js]: not sure this is true - there are other internal asru roles which aren't licensing or inspector.

Comment (wheelsandcogs) [reply lib/hooks/create/project.js]: Yeah but don't we want an ASRU admin / ASRU basic revocation to go through licensing as well? Should I just correct the comment?

Comment (joefitter) [reply lib/hooks/create/project.js]: that's what I was getting at :) it's just slightly misleading and not immediately obvious what's happening from a glance

Comment (wheelsandcogs) [reply lib/hooks/create/project.js]: 👍

**Pull request merged**